.github/workflows/nomad.yaml

```
@@ -131,41 +130,66 @@ jobs:
+    outputs:
+      result: "${{ steps.read.outputs.result }}"
+
   wait-for-all-jobs:
     name: Wait for all jobs to finish
     runs-on: [self-hosted, wind-tunnel]
     needs: run-scenarios
     steps:
       - uses: actions/checkout@v5
 
+      - name: Print matrix json
+        run: |
```

actionlint (1.7.8)

property "read" is not defined in object type {run-scenarios: {outputs: {}; result: string}}

(expression)